scripts/install-requirements.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
```

Comment:
Maybe we can include this script as dependency-installer script in the plugin, and if this file is present execute it during the installation?

Reply:
Or we can use #3018 and problem solved

Reply:
> Or we can use #3018 and problem solved

Different problems, this is for execute a script installation for plugin

Reply:
Yes but with deb packages it auto installs all those dependencies on linux.